### ISSUE TITLE
Set Object Expiration for reports in the S3 bucket

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/va.gov-team#29609]
As a follow up to: https://app.zenhub.com/workspace/o/department-of-veterans-affairs/va.gov-team/issues/28265

Now that we have gotten the reports into a stable environment in S3 (both E2E Testing reports and A11y Scan reports), we need to set Object Expiration timers for both the reports and the videos of failed tests to ensure that the S3 bucket does not balloon too large over time. The following steps need to be taken:

1) File an ops ticket for permissions to both be able to delete files and to be able to manage object expiration settings
2) Set Object Expiration settings to 30 days for both E2E test reports and Videos.  We will also start A11y test reports at 30 days, however those may change after consulting with the powers that be.
3) Delete any files that have made it where they weren't supposed to during testing to keep the S3 bucket easy to manage over time.

**Comment by CBonade** [comment]
Object expiration has been set on this bucket by ops. Still awaiting permissions to manage these options ourselves, but for now everything will auto-delete as necessary.

**Comment by pjhill** [comment]
Done!
